templates/home.html

```diff
@@ -19,19 +19,16 @@
                     </p>
                     <ul class="menu-list">
                         <li><a href="{{url_for('logout')}}">Log Out</a></li>
-                        {% if is_subscribed %}
+                        <!-- {% if is_subscribed %}
                             <li><a href="{{url_for('unsubscribe')}}">Unsubscribe</a></li>
                         {% else %}
                             <li><a href="{{url_for('subscribe')}}">subscribe</a></li>
-                        {% endif %}
+                        {% endif %} -->
```

Comment:
Intentional change or stray comment?

Reply:
Intentional change -- should be an open comment a few lines up. That said I'm open to putting it back in.

Currently if you unsubscribe there's no easy/natural way to resubscribe (I suppose you could get a new "enroll" link and it would snap you back into the home page, but that' not great). The difference between "enrolled" and "subscribed" was a side-effect of the auth0 account system.

I wanted to keep the code around since I figured we might want to bring it back or need to have easy access to it, but my gut also said "this isn't right anymore" 

I think there's a larger group discussion to have about the various flavors of "stop emailing me" and their systematic meanings which is the "Correct" way to resolve this feeling.

Reply:
Makes sense! Thanks for the explanation